Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UnityPlayerBuildTools.cs

```diff
@@ -212,8 +213,21 @@ public static void ParseBuildCommandLine(ref IBuildInfo buildInfo)
                     case "-autoIncrement":
                         buildInfo.AutoIncrement = true;
                         break;
-                    case "-scenes":
-                        // TODO parse json scene list and set them.
+                    case "-sceneList":
+                        buildInfo.Scenes = buildInfo.Scenes.Union(from scene in arguments[++i].Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
+                                                                  select scene.Trim());
+                        break;
+                    case "-sceneListFile":
+                        string path = arguments[++i];
+                        if (File.Exists(path))
+                        {
+                            buildInfo.Scenes = buildInfo.Scenes.Union(from scene in File.ReadAllText(path).Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Nit: scene.Trim()

Reply:
Nit: Also would be nice to pull out the text parsing to a function so these two long lines aren't mostly duplicated code.

Reply:
Strange, how'd I reintroduce that? I really should do a better pass before pushing.